l10n_th_withholding_tax_report/data/report_data.xml

```diff
@@ -18,14 +18,15 @@
         file="l10n_th_withholding_tax_report.report_withholding_tax_qweb"
         paperformat='l10n_th_withholding_tax_report.report_qweb_paperformat'
     />
-    <!-- CSV REPORTS -->
+    <!-- TEXT REPORTS -->
     <report
-        id="action_withholding_tax_report_csv"
+        id="action_withholding_tax_report_txt"
         model="withholding.tax.report"
-        string="Withholding Tax Report (CSV)"
-        report_type="csv"
-        name="report_withholding_tax_csv"
-        file="report_withholding_tax_csv"
+        string="Withholding Tax Report (TXT)"
+        report_type="qweb-text"
+        name="l10n_th_withholding_tax_report.report_withholding_tax_text"
+        file="l10n_th_withholding_tax_report.report_withholding_tax_text"
+        print_report_name="'%sOZZZZZ%s%s0000' % (object.company_id.vat, object.format_pnd(object.income_tax_form), object.format_date())"
```

Comment:
Change to more readable name. I..e, WHT-xxx-yyy-zzz (I don't know, you can suggest).